### PR TITLE
Remove owner migrations from keeper registry

### PR DESCRIPTION
**Pull request by RyanRHall**

### CODE REVIEW

File: contracts/src/v0.8/interfaces/MigratableKeeperRegistryInterface.sol [hunk 1]
```diff
@@ -5,9 +5,22 @@ pragma solidity ^0.8.0;
 import "../UpkeepFormat.sol";
 
 interface MigratableKeeperRegistryInterface {
+  /**
+   * @notice Migrates upkeeps from one registry to another, including LINK and upkeep params.
+   * Only callable by the upkeep admin. All upkeps must have the same admin. Can only migrate active upkeeps.
```
Comment (FelixFan1992) [inline contracts/src/v0.8/interfaces/MigratableKeeperRegistryInterface.sol:10]: All upke**e**ps

File: contracts/test/v0.8/KeeperRegistry.test.ts [hunk 2]
```diff
@@ -2066,6 +2066,12 @@ describe('KeeperRegistry', () => {
           .to.emit(registry2, 'UpkeepReceived')
           .withArgs(id, toWei('100'), registry.address)
       })
+      it('is only migratable by the admin', async () => {
```
[the remark this answers is not in the document]
Comment (se3000) [reply contracts/test/v0.8/KeeperRegistry.test.ts:2069]: Oh, or make a story to use this: https://github.com/dangreenisrael/eslint-plugin-jest-formatting/blob/master/docs/rules/padding-around-test-blocks.md

[the remark this answers is not in the document]
Comment (RyanRHall) [reply contracts/test/v0.8/KeeperRegistry.test.ts:2069]: [done!](https://app.shortcut.com/chainlinklabs/story/42557)

File: contracts/src/v0.8/interfaces/MigratableKeeperRegistryInterface.sol (hunk 1 above)
Comment (RyanRHall) [reply contracts/src/v0.8/interfaces/MigratableKeeperRegistryInterface.sol]: fixed!

**Review by FelixFan1992 — APPROVED** [review]

**Review by infiloop2 — APPROVED** [review]

**Review by infiloop2 — APPROVED** [review]

**Review by kaleofduty — APPROVED** [review]

**Review by payephyoeaung87 — COMMENTED** [review]
Hello everyone, I'm not sure if this is a good idea, but it's a good idea.This problem is entirely about me.I'm very upset about this at remix@ethereum.org. In fact, my smart contract was bridged by Binance to MetaMask Wallet. Finally, I reached the ENDpoint and my assets changed to ETH.And I lost my connection due to a communication problem in my country. If you do not believe me, I have a key that contains a lot of ETHs.My assets are also stored in the remix-etherscan-plugin. Welcome to Remix 0.24.1


 Your files are stored in indexedDB, 11.26 MB / 30.94 GB used


 You can use this terminal to:

 Check transactions details and start debugging.

 Execute JavaScript scripts:
 - Input a script directly in the command line interface
 - Select a Javascript file in the file explorer and then run \ `remix.execute () \` or \ `remix.exeCurrent () \` in the command line interface
 Right click on a JavaScript file in the file explorer and then click \ Run

 The following libraries are accessible:
 remix

 Type the library name to see available commands. What can I do next?

**Pull request merged**